server/integration_tests/test_data/place_detection_e2e_dc/query_6/chart_config.json

```diff
@@ -884,14 +826,6 @@
           "sdg/SI_POV_DAY1": {
             "name": "Proportion of population below international poverty line",
             "statVar": "sdg/SI_POV_DAY1"
-          },
-          "sdg/SP_ACS_BSRVH2O": {
```

Comment:
these svs are pushed out because the topic they are part of are no longer expanded because of a newly added topic (dc/topic/sdg_1.2) that is expanded instead